### DISCUSSION TITLE
Cant see Github Pro badge and Developer Program Member badge

### DISCUSSION THREAD

**Discussion by crizmo** [community/community#41490] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello for more than a month now i cant see the pro badge under my profile even tho i have pro enabled .
Same goes for developer program member . The badge is visible from my mobile but i cant see it on the website

Plus i don't see any checkbox for it if i click the "edit profile" button.
Some of my friends who got the pro badge the same way i did from filling the student developer pack still have it while mine is gone

**Comment by DSDmark** [comment]
Hi @crizmo,

Did you update your **preferences**?

![image](https://user-images.githubusercontent.com/85744816/207037876-7f303cac-1592-4dc9-a572-5ee12823fb3b.png)

Also, you could try these steps. Maybe its work for you:-

- Clear your browser's cache and cookies. Sometimes, outdated information stored in your browser's cache can cause issues with displaying the GitHub Pro badge on your profile. Clearing your cache and cookies can help resolve this issue.

- Try accessing your profile from a different web browser. If the issue persists, it may be a problem with your browser. Try accessing your profile from a different web browser to see if the issue is specific to the browser you were using previously.

If it's not the case. Then, It sounds like you may not have a GitHub Pro subscription. 

And the final things you could do. If you think, you have nothing for it. Contact GitHub support:- [**GitHub Support**](https://support.github.com/)

**Reply by DSDmark** [reply]
And also I forget to mention inside the settings. There is a billing section. You may find some info about pro badge. So, don't forget to check out that.

**Reply by crizmo** [reply]
Hey It seems the whole Highlights tab is missing from my profile
thats why both pro and  developer program member are missing

**Comment by crizmo** [comment, marked as answer]
Thanks for replying

The thing is i don't have that option
![image](https://user-images.githubusercontent.com/83665497/207051753-8ce8ed8f-8e16-49da-a073-6ba90c073adb.png)

Nor is there any button to enable them here
![image](https://user-images.githubusercontent.com/83665497/207053156-6964cd5b-7370-489a-8291-a7169980cdcd.png)

Plus my GitHub copilot and pages are still active.
Even on the student pack page it shows i still have the plan active .

Both the pro badge and developer program member are not visible from any browser even after clearing cache and cookies.
Earlier i used to see both of them but they just stopped showing now

Ig ill just contact the support . Thanks !
